src/mpyl/cli/commands/build/__init__.py

```diff
@@ -1 +1,21 @@
 """CLI support for Jenkins multibranch pipeline"""
+import os
+import shutil
+from dataclasses import dataclass
+from enum import Enum
+
+from rich.console import Console
+
+
+@dataclass(frozen=True)
+class Sound(Enum):
+    SUCCESS = 'Glass.aiff'
+    FAILURE = 'Sosumi.aiff'
+
+
+def play_sound(sound: Sound):
+    if shutil.which('afplay') is None:
+        Console().bell()
+        return
```

Comment:
Maybe it's more clear to check for the type of platform mpyl is running on?
There's the `platform` package with `platform.system()` and also `os.name`.